connectors/src/connectors/slack/lib/slack_client.ts

```diff
@@ -52,6 +52,22 @@ export async function getSlackClient(
         // @ts-expect-error can't get typescript to be happy with this, but it works.
         return await Reflect.apply(target, thisArg, argumentsList);
       } catch (e) {
+        // If we get rate limited, we throw a known error. The activity will be
```

Comment:
I don't think we should mention the activity logic here, for a few reasons:
The function `getSlackClient()` can be called outside of an activity, and the activity logic should not be documented here. It will quickly get out of sync.